netlify.toml

```diff
@@ -0,0 +1,17 @@
+# This configuration is for the standalone version of blockbuilder-search.
+# Used with netlify-cli tool
+# https://www.netlify.com/docs/netlify-toml-reference/
+[build]
+  command = 'npm run parcel-build'
+  publish = 'dist'
+
+[dev]
+  command = 'npm run parcel'
+  port = 1234
+  publish = "dist"
+
+[[redirects]]
+  from = "/api/*"
+  to = "https://blockbuilder.org/api/:splat"
+  status = 200
+  force = true
```

Comment:
~we do this now from the **blockbuilder** editor project.  (where this === point to the production search backend for local development by default)~ edit I do this manually by editing my `secrets.json` file in that project.

 nice thinking taking this idea one step farther and  removing the need to run the **blockbuilder** editor project.